### ISSUE TITLE
OpenAI gym windows hangs on render

### ISSUE THREAD

**Issue by agrawal-rohit** [openai/gym#803]
`import gym
import random
import numpy as np
import keras
from keras.layers import Dense, Dropout
from keras.models import Sequential
from keras.wrappers.scikit_learn import KerasRegressor
from statistics import mean,median
from collections import Counter

LR = 1e-3 # Learning Rate
env = gym.make('CartPole-v0')
env.reset()
goal_steps = 500
score_requirement = 50
initial_games = 10000

def some_random_games_first():
    for episode in range(5):
        env.reset()
        for t in range(goal_steps):
            env.render()
            action = env.action_space.sample() # Generates a sample action
            observation, reward, done, info = env.step(action)
            if done:
                break
           
some_random_games_first()            `

This is the code I am trying to run. I am using Anaconda's Spyder, Python 3.6. As soon as I compile, The environment window shows up but it's unresponsive and could only be closed using the task manager. The pyglet version is 1.2.4 ( I don't know it matters or not ).

**Comment by cristyandavid1** [comment]
In my opinion we are doing everything wrong.
 We live in a real world. Not a matrix.
You want a bot to be great
Give me a gym, and I’ll be your agent
Working with a A.i.. with the smart glasses

**Comment by cristyandavid1** [comment]
We humans have the perfect body for learning.

**Comment by supratikp** [comment]
Did you manage to fix this?
